mssqlcli/mssqlqueries.py

```diff
@@ -90,8 +90,7 @@ def get_views():
     return '''
         SELECT  table_schema,
                 table_name
-        FROM INFORMATION_SCHEMA.TABLES
-        WHERE table_type = 'View'
+        FROM INFORMATION_SCHEMA.VIEWS
```

Comment:
My guess as to why this isn't working is you are running against a database with case-sensitive collation, so 'View' doesn't match any rows.

Reply:
I see, can we change it to `INFORMATION_SCHEMA.VIEWS` so that we don't need to worry about casing?

Reply:
I'm fine with the change to VIEWS